internal/capability/operator_install.go

```diff
@@ -18,11 +19,13 @@ func OperatorInstallAllFromCatalog(catalogSource string, catalogSourceNamespace
 
 	s, err := operator.Subscriptions(catalogSource, catalogSourceNamespace)
 	if err != nil {
+		logger.Errorf("%s", err)
```

Comment:
How about just returning the error and printing it at the upper level call? But enriching the error with which function generated it? And maybe allowing for a debug log not an Errorf one, that when activated, can drop all the state on the screen in this function. This comment applies to multiple instances of the same type of log.